### PR TITLE
Update actions for `staging` rename and document branching flow

### LINKED ISSUE #3: Set up frontend repo
Definition of done:

- There is a create-react-app skeleton.
- There is rudimentary CI (npm run test on PRs to main)
- There is basic deployment. (Git pushes can trigger deploy to staging and prod envs.)

### PR DESCRIPTION
**Pull request by WillEngler**
Closes #3.

# Summary

Updates GitHub actions that CI triggers on PRs to staging. And pushes to staging trigger deploys to the staging env.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,21 @@
...
+- `prod` is a branch used to deploy vetted code from the `staging` branch to production, https://thegardens.ai/. 
+
+The `staging` branch is the source of truth for development in this repo. `prod` only exists for deployment purposes. If you make a change to `prod` that isn't on `staging`, it will soon get blown away. If there are merge conflicts between `prod` and `staging`, we will force push the contents of `staging` to `prod` with `git push origin --force staging:prod`.
+
+# Workflow
+
+1. When you pick up a feature or bug, make a branch off of staging. Like `will/45-new-page`.
+2. Make the needed changes on your branch and test them locally.
+3. Open a PR from your branch to `staging`.
+4. If the tests pass and another developer approves your change, merge in to staging. After merging, see if everything
```
Comment (kjschmidt913) [inline README.md]: I think you forgot to finish your thought here

File: README.md [hunk 2]
```diff
@@ -1,6 +1,21 @@
...
+
+# Workflow
+
+1. When you pick up a feature or bug, make a branch off of staging. Like `will/45-new-page`.
+2. Make the needed changes on your branch and test them locally.
+3. Open a PR from your branch to `staging`.
+4. If the tests pass and another developer approves your change, merge in to staging. After merging, see if everything 
+5. If so, open a "deployment" PR from staging into prod and merge it
+
 # Getting Started with Create React App
```
Comment (kjschmidt913) [inline README.md:23]: Should we add the getting started commands here? Like do an `npm install` and then...is it `npm start` to run it?

**Review by kjschmidt913 — CHANGES_REQUESTED** [review]
Looks good! I think just those 2 things need to be added to the readme. Thanks for putting this together!

File: README.md (hunk 1 above)
Comment (WillEngler) [reply README.md]: lol dang. Fixed 😄

File: README.md (hunk 2 above)
Comment (WillEngler) [reply README.md:23]: Sure, went ahead and added

**Review by kjschmidt913 — APPROVED** [review]

**Pull request merged**